src/test/java/io/cdap/plugin/salesforce/SalesforceSchemaUtilTest.java

```diff
@@ -244,4 +250,35 @@ private Field getFieldWithType(String name, FieldType type, boolean isNillable)
 
     return field;
   }
+  
+  @Test
+  public void testSourceSchemaNotNullIfConnectionMacroAndImportManually() {
+    Schema schema = Schema.recordOf("output",
+                                    Schema.Field.of("Id",
+                                                    Schema.of(Schema.Type.STRING)));
+    SalesforceSourceConfig mockConfig = Mockito.mock(SalesforceSourceConfig.class);
+    MockPipelineConfigurer mockPipelineConfigurer = new MockPipelineConfigurer(null);
+    SalesforceBatchSource source = new SalesforceBatchSource(mockConfig);
```

Comment:
Please add a test for multi source.

Reply:
Added. But for Multisource, output schema will always be null.